### ISSUE TITLE
Add ability to only return status code

### ISSUE THREAD

**Issue by jamesmallen** [mwarkentin/django-watchman#113]
A very useful feature for us would be the ability to have blank responses with HTTP status codes if the user is anonymous, and the normal operation if the user attempts authenticationf.

One way to do this would be to add a setting `WATCHMAN_ANONYMOUS_DETAILS` with a default value of `True`. I'm happy to put together a PR exploring this possibility

**Comment by mwarkentin** [comment]
Hey @jamesmallen, thanks for using django-watchman and suggesting this improvement. :)

I was wondering if you could provide a bit more info on your use case for the anonymous vs. authenticated checks? Would you expect to use both types?

I'm wondering whether a blank response would be the right way to go or if it would just be to restrict the responses to "ok: true/false" and disable the other pieces like stack traces?

**Comment by jamesmallen** [comment]
On second thought, there's a lot of ambiguity around behavior when this is introduced. I'm wondering if having another view, that can be used instead of or in addition to the default urlpatterns might be a good way of achieving our goal. We would prefer not to reveal any implementation details for the anonymous view

**Comment by jamesmallen** [comment]
I just opened a PR to implement the "bare status" endpoint. It is not part of the default `urls`, as it seemed to be an edge case. The PR also includes tests and an update to the README to show potential usage.

To avoid code duplication, I refactored the views to use a helper function to generate the status object. Let me know if you have any thoughts or suggestions about different ways to approach this!
